snippet-extractor-output/snippets.yaml

```diff
@@ -11153,6 +11153,43 @@
         for (var i = 0; i < properties.items.length; i++)
             console.log("Property Name:" + properties.items[i].key + "; Type=" + properties.items[i].type + "; Property Value=" + properties.items[i].value);
     });
+'Word.Document#changeTrackingMode:member':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/99-preview-apis/manage-change-tracking.yaml
+
+    await Word.run(async (context) => {
+      const document = context.document;
+      document.load("changeTrackingMode");
+      await context.sync();
+
+      if (document.changeTrackingMode === Word.ChangeTrackingMode.trackMineOnly) {
```

Comment:
Consider adding this snippet to `Word.ChangeTrackingMode:enum` as well.

Reply:
Do you know a way to add that? If not, we may need to update the script.

Reply:
I'll look into that

Reply:
Thanks for addressing. I've incorporated